linkerd/meshtls/rustls/src/creds/verify.rs

```diff
@@ -0,0 +1,56 @@
+use linkerd_identity as id;
+use std::convert::TryFrom;
+use std::time::SystemTime;
+use std::{io, sync::Arc};
+use tokio_rustls::rustls::{
+    self,
+    client::{self, ServerCertVerified, ServerCertVerifier},
+    server::ParsedCertificate,
+    Certificate, RootCertStore, ServerName,
+};
+use tracing::trace;
+
+pub(crate) struct AnySANVerifier(Arc<RootCertStore>);
```

Comment:
nitpicky, but the general Rust convention is that acronyms in type names should only have the first letter capitalized (see https://rust-lang.github.io/api-guidelines/naming.html#casing-conforms-to-rfc-430-c-case); so this should be
```suggestion
pub(crate) struct AnySanVerifier(Arc<RootCertStore>);
```